presto-testing-docker/src/main/java/io/prestosql/testing/docker/DockerContainer.java

```diff
@@ -69,16 +70,17 @@ public DockerContainer(String image, List<Integer> ports, Map<String, String> en
         this.image = requireNonNull(image, "image is null");
         this.environment = ImmutableMap.copyOf(requireNonNull(environment, "environment is null"));
         this.capabilities = ImmutableList.copyOf(requireNonNull(capabilities, "capabilities is null"));
+        this.healthCheck = requireNonNull(healthCheck, "healthCheck is null");
         try {
-            startContainer(ports, healthCheck);
+            startContainer(ports);
```

Comment:
> `Store healthCheck as field`

nit: why not store `ports` as a field too?

Reply:
because it is already stored as `hostPorts` (map)